### PR TITLE
Require statement info updated

### PR DESCRIPTION
**Pull request by Aniket-Engg**

### CODE REVIEW

File: docs/running_js_scripts.md [hunk 1]
```diff
@@ -1,13 +1,44 @@
-Running JS Scripts in Remix
+Running JS Scripts
 ===========================
 
-Remix accepts async/await scripts to run [web3.js](https://web3js.readthedocs.io/)  or [ethers.js](https://docs.ethers.io/) commands. The script needs to be wrapped in a self executing function.
+_JavaScript (JS) is a lightweight, interpreted, or just-in-time compiled programming language with first-class functions._
```
Comment (ryestew) [inline docs/running_js_scripts.md:4]: I"m not sure this helps - everyone knows what JS is - or is there another reason you are putting this in?

File: docs/running_js_scripts.md [hunk 2]
```diff
@@ -60,3 +85,11 @@ For more information about this example, please see: [running async/await script
 ```
 
 For more script examples, please see [Frequently Asked Scripts](FAS.html).
+
+## `require` in scripts at Remix
+
+`require` statement is supported in limited manner for remix supported modules with Remix Scripts.
```
Comment (ryestew) [inline docs/running_js_scripts.md:91]: the sentence above should be: (I added an a before limited)(and changed the case of Remix)
`require` statement is supported in a limited manner for Remix supported modules with Remix Scripts.

File: docs/running_js_scripts.md (hunk 1 above)
Comment (Aniket-Engg) [reply docs/running_js_scripts.md:4]: This doesn't seem an issue. I would like to keep it.

File: docs/running_js_scripts.md (hunk 2 above)
Comment (Aniket-Engg) [reply docs/running_js_scripts.md]: updated

**Review by ryestew — APPROVED** [review]

**Pull request merged**